docs/content/direct/deploy-on-k3d.md

```diff
@@ -0,0 +1,142 @@
+# Deploy KubeStellar on K3D
+
+This document shows how to deploy kubestellar v0.21.0 and a managed cluster on K3D
+
+## Prereqs
+
+In addition to [pre-reqs](pre-reqs.md), install k3d v5.6.0 (only k3d version tested so far)
+
+## Common Setup for standard examples
+
+1. You may want to `set -e` in your shell so that any failures in the setup or usage scenarios are not lost.
+
+1. If you previously installed KS on K3D:
+    ```shell
+    k3d cluster delete kubeflex
+    k3d cluster delete wec1
+    kubectl config delete-context kubeflex || true
+    kubectl config delete-context wec1 || true
+    ```
+   If previously running KS on Kind, clean that up with [the Kind cleanup script](../../../test/e2e/common/cleanup.sh).
+
+1. Set environment variables to hold KubeStellar and OCM-status-addon desired versions:
+    ```shell
+    export KUBESTELLAR_VERSION=0.21.0
+    export OCM_STATUS_ADDON_VERSION=0.2.0-rc6
+    ```
+
+1. Create a K3D hosting cluster with nginx ingress controller:
+    ```shell
+    k3d cluster create -p "9443:443@loadbalancer" --k3s-arg "--disable=traefik@server:*" kubeflex
+    helm install ingress-nginx ingress-nginx --repo https://kubernetes.github.io/ingress-nginx --version 4.6.1 --namespace ingress-nginx --create-namespace
+    kubectl config rename-context k3d-kubeflex kubeflex
+    ```
+
+1. [Paolo says] when we use kind, the name of the container is kubeflex-control-plane and that is what we use 
```

Comment:
[Paolo..] This should be remove on public doc ..

Reply:
Done